src/sumo/wrapper/sumo_client.py

```diff
@@ -401,6 +402,47 @@ def _delete():
 
         return retryer(_delete)
 
+    def _get_retry_details(self, response_in):
+        assert response_in.status_code == 202, (
+            "Incorrect status code; expcted 202"
+        )
+        headers = response_in.headers
+        location = headers.get("location")
+        assert location is not None, "Missing header: Location"
+        assert location.startswith(self.base_url)
+        retry_after = headers.get("retry-after")
+        assert retry_after is not None, "Missing header: Retry-After"
+        location = location[len(self.base_url) :]
+        retry_after = int(retry_after)
+        retry_after = 10
```

Comment:
Overwriting

Reply:
Good catch... I'll remove that. We should probably change the `retry-after` settings in `sumo-core`... it is currently 60 seconds, but 15 seconds may be a more reasonable value.